rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -634,6 +634,40 @@ if should_install_app "joininbox" ; then
     fi
 fi
 
+# Install JoinMarket GUI
+if should_install_app "jm"; then
+    if [ $IS_RASPI = 1 ] || [ $IS_X86 = 1 ]; then
+        JMG_UPGRADE_URL=https://github.com/manasgandy/joinmarket-gui/archive/refs/tags/%JMG_VERSION.tar.gz
```

Comment:
Does this need to be $JMG_VERSION rather than %JMG_VERSION?

Reply:
ya, my bad